packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -106,11 +104,70 @@ function isExpired<T>(item: CacheEntry<T>): boolean {
   if (item.expires === undefined) {
     return false;
   }
-  return !item.expires.isAfter(moment());
+  // return !item.expires.isAfter(moment());
+  return item.expires < Date.now();
 }
 
 function inferExpirationTime(
-  expirationTime: moment.MomentInputObject | undefined
-): moment.Moment | undefined {
-  return expirationTime ? moment().add(expirationTime) : undefined;
+  expirationTime: DateInputObject | undefined
+): number | undefined {
+  return expirationTime ? inferExpirationTimeFromDate(expirationTime) : undefined;
+}
+
+interface DateInputObject {
```

Comment:
As I see it this is our API and we use it like: 
```new Cache<Destination>({ hours: 0, minutes: 5, seconds: 0 });```
So I would also only support times up to hours. I would also only allow for one keyword for one time property and pick the one with the plural `s` i.e. `hours,minutes,seconds`

Reply:
@marikaner @jjtang1985 @tomfrenken what is your opinion on this?

Reply:
I actually suggested to use milliseconds and just a number, so that the default consistent with the specific expiration times (which are also just a number and ms). We agreed to do this on a follow up.